Resources/Prototypes/_LostParadise/Datasets/nightmare.yml

```
@@ -0,0 +1,307 @@
+- type: dataset
+  id: nightmare
+  values:
+    #ss13
+  - U'ruan
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)